core/src/main/scala/zio/config/ConfigDescriptor.scala

```diff
@@ -170,15 +171,66 @@ object ConfigDescriptor {
   val float: ConfigDescriptor[String, String, Float] =
     ConfigDescriptor.Source(ConfigSource.empty, PropertyType.FloatType) ?? "value of type float"
 
+  def first[K, V, A](path: K)(desc: ConfigDescriptor[K, V, A]) =
```

Comment:
Probably `head` would be more consistent with Scala, since it's the head of a `list`. Also we should have the variant without `path`, like for the other descriptors.

Reply:
done